census/views/_snippets/form.html

```diff
@@ -1,12 +1,12 @@
 {% macro currentEntryValue(qu) %}
 {% if current %}
-{% if current.answers[qu.id] %}
-{{current.answers[qu.id]}}
+  {% if current.answers[qu.id] %}
+    {{current.answers[qu.id]}}
+  {% else %}
+    {{ gettext("No entry") }}
+  {% endif %}
```

Comment:
This could be simplified to:
```
{% if current and current.answers[qu.id] %}
  {{ current.answers[qu.id] }}
{% else %}
  {{ gettext("No entry") }}
{% endif %}
```

Also note the pattern of adding spaces between `{{` and `}}` (as in `{{ foo }}` instead of `{{foo}}`)